PyRDF/Node.py

```diff
@@ -22,27 +28,46 @@ def __init__(self, _get_head, operation):
         self.value = None
 
     def __getattr__(self, attr):
-        self._cur_attr = attr
+        # Interception of operation calls
```

Comment:
Make it a proper docstring comment and elaborate a bit more: "Intercepts any call to an operation node and dispatches it by means of a call handler."